main.go

```diff
@@ -187,7 +188,7 @@ func main() {
 
 	flag.StringVar(&nodes, "nodes", "127.0.0.1", "nodes")
 	flag.BoolVar(&clientCompression, "client-compression", true, "use compression for client-coordinator communication")
-	flag.IntVar(&concurrency, "concurrency", 16, "number of used goroutines")
+	flag.IntVar(&concurrency, "concurrency", runtime.GOMAXPROCS(0), "number of used goroutines")
```

Comment:
Why not runtime.NumCPU()?

Reply:
Yeah, `runtime.NumCPU` is a better fit. Changed.